dissect/target/helpers/keychain.py

```diff
@@ -58,7 +64,7 @@ def parse_key_type(key_type_name: str) -> KeyType:
 
 def register_wildcard_value(value: str) -> None:
     for key_type in KeyType:
-        register_key(key_type, value)
+        register_key(key_type, value, is_wildcard=True)
 
 
 def register_keychain_file(keychain_path: Path) -> None:
```

Comment:
Not sure. But dont you want to register wildcard values through keychain files as well? I think this can't be done like this currently. Correct me when im wrong though.

Reply:
Seems like this currently not possible. Should be done at a later time, through a different PR.

Reply:
Wildcard values from the keychain file are currently not allowed, but this is something we could add in a future PR.